frontend/sw360-portlet/src/main/java/org/eclipse/sw360/portal/tags/OutTag.java

```diff
@@ -86,6 +88,11 @@ protected String escapeInSingleQuote(String value) {
         return value.replaceAll("'", quoteReplacement("\\\'"));
     }
 
+    private String prepareTitleAttribute(String value) {
```

Comment:
I didn't test this thoroughly, but since the problem stems from the JS code, I think a simple
`StringEscapeUtils.escapeJavaScript(value)` might be enough.  Then you could get rid of  `escapeInDoubleQuote` and `prepareTitleAttribute` alltogether.  I quickly implemented it and it seemed to work.
This might be more foolproof than just replacing a few characters.

However, I am not sure, if this leads to problems elsewhere....

Reply:
It seems that this approach leads to problems with the `<sw360:ProjectName>`  as this inherits from the `<sw360:out>` tag....

But there also is an `<sw360:DisplayDescription>` tag.  Maybe one should use this on the homepage.

Reply:
What about to remove the ´<sw360:DisplayDescription>´ tag? This tag is not really necessary and duplicated code. Instead of ´<sw360:DisplayDescription>´ we can use the normal ´<sw360:out>´ tag. 
I also removed the ´ escapeInDoubleQuote´ and ´ escapeInSingleQuote´ functions. I think also its better to use ´SringEscapeUtils.escapeJavaScript()´

The issue regarding ProjectName and ComponentName is solved by decoding HTML entities.

Id did a lot of tests and everything looks quite good now.

Reply:
Sounds all good to me! :)